src/components/Tabs/NotesComponent.tsx

```diff
@@ -0,0 +1,181 @@
+import {
+  ChakraProvider,
+  Tabs,
+  TabList,
+  TabPanels,
+  Tab,
+  TabPanel,
+} from "@chakra-ui/react";
+import theme from "../ui/tabsTheme";
+import { useState } from "react";
+import { Input } from "../ui/input";
+import { Button } from "../ui/button";
+import axios from "axios";
+import { CheckIcon, EditIcon, TrashIcon } from "lucide-react";
+import { useQuery } from "@tanstack/react-query";
+import { INote } from "@/server/db/models/UserModel";
+import { IGame } from "@/server/db/models/GameModel";
+import { TextArea } from "../ui/textarea";
+
+function formatDate(date: Date) {
+  let day = date.getDate().toString().padStart(2, "0");
+  let month = (date.getMonth() + 1).toString().padStart(2, "0");
+  let year = date.getFullYear();
+  return month + " - " + day + " - " + year;
+}
+
+interface NotesComponentProps {
+  userId: string;
+  gameData: IGame & { _id: string };
+}
+
+export default function NotesComponent({
+  gameData,
+  userId,
+}: NotesComponentProps) {
+  const [newNote, setNewNote] = useState("");
+  const [editNoteId, setEditNoteId] = useState("");
+  const [editNote, setEditNote] = useState("");
+
+  const { data: notes, refetch } = useQuery({
+    queryKey: ["notes"],
+    queryFn: async () => {
+      const response = await fetch(
+        `/api/users/${userId}/notes?gameId=${gameData._id}`,
+      );
+      if (!response.ok) {
+        throw new Error("Failed to fetch notes");
+      }
+      const data = await response.json();
+      return data;
+    },
+    retry: 3,
+  });
+
+  async function handleAddNote() {
+    await axios.post(
+      `/api/users/${userId}/notes`,
+      JSON.stringify({
+        date: new Date(),
+        description: newNote,
+        gameId: gameData._id,
+      }),
+      {
+        headers: {
+          "Content-Type": "text",
+        },
+      },
+    );
+
+    setNewNote("");
+    await refetch();
+  }
+
+  async function handleDeleteNote(noteId: string) {
+    await axios.delete(`/api/users/${userId}/notes/${noteId}`, {
+      headers: {
+        "Content-Type": "text",
+      },
+    });
+
+    await refetch();
+  }
+
+  async function handleEditNote() {
+    await axios.put(
+      `/api/users/${userId}/notes/${editNoteId}`,
+      JSON.stringify({
+        date: new Date(),
+        description: editNote,
+        gameId: gameData._id,
+      }),
+      {
+        headers: {
+          "Content-Type": "text",
+        },
+      },
+    );
+
+    await refetch();
+    setEditNoteId("");
+  }
+
+  return (
+    <ChakraProvider theme={theme}>
```

Comment:
Lauren is moving this to _app and centralizing all modifications to chakra theme config to `src/styles/theme.tsx` (can you do the same or maybe wait for her PR to get merged)